### PR TITLE
feat: Spec and Doc fixes for Release 1.4.0

### PR DESCRIPTION
**Pull request by kevinshahfws**
This PR contains below fixes:

1. Add an array of suggested entities to the Search Intent
2. Update device.distributor description
3. Update device.platform description
4. Update device.model description
5. Update device.make description
6. Update Localization.AdditionalInfo
7. Update capability for SecondScreen.protocols
8. Remove extra space from Discovery.launch example

### CODE REVIEW

**Review by jlacivita — CHANGES_REQUESTED** [review]

File: src/schemas/intents.json [hunk 1]
```diff
@@ -609,6 +609,33 @@
...
+                                },
+                                "suggestions": {
+                                    "type": "array",
+                                    "properties": {
+                                        "entity": {
+                                            "type": "object",
+                                            "items": {
+                                                "anyOf": [
+                                                    {
+                                                        "$ref": "https://meta.comcast.com/firebolt/entity#/definitions/MovieEntity"
```
Comment (kpears201) [inline src/schemas/intents.json:621]: You can use `ProgramEntity` which is an already defined list of these entities.
But how come searching can only suggest programs? Seems like Channels and Music makes sense to suggest too?

File: src/openrpc/discovery.json [hunk 2]
```diff
@@ -1250,6 +1250,43 @@
...
+            {
+              "name": "appId",
+              "value": "xrn:firebolt:application-type:main"
+            },
+            {
+              "name": "intent",
+              "value": {
+                "action": "search",
+                "data": {
+                  "query": "a cool show",
```
Comment (jlacivita) [inline src/openrpc/discovery.json:1266]: I guess we should maybe search for "a cool movie" given the results...

Comment (jlacivita) [reply src/openrpc/discovery.json:1266]: sorry, I know I sent you that example :)

**Review by jlacivita — DISMISSED** [review]
Looks good

**Review by kpears201 — DISMISSED** [review]

**Review by jlacivita — DISMISSED** [review]

**Review by jlacivita — DISMISSED** [review]

**Review by jlacivita — APPROVED** [review]

**Pull request merged**